### ISSUE TITLE
Tooltips get cutoff when used in horizontal table

### ISSUE THREAD

**Issue by ErikSchierboom** [Chalarangelo/mini.css#169]
* Framework Version: 3.0.1
* Flavor used: default
* Operating System/Browser: Chrome/Mac OS
* Code to reproduce: https://codepen.io/erikschierboom/pen/ZEEPWdw

Reproduce by hovering over the "Text #1.1", "Text #1.2", or "Text #2.1" texts inside the table.

* Screenshots:

![image](https://user-images.githubusercontent.com/135246/69259559-bf560980-0bbe-11ea-9e5d-87018a5c9d75.png)

* Bug description:

When using tooltips in a horizontal table, the tooltips are cutoff on the border of the table. I'd like for the tooltips to be displayed in their entirety.

**Comment by fidgol** [comment]
Same issue here. I also tried to experiment with different z-index but hat no success.
Any update on this issue?

**Comment by sigmasoldi3r** [comment]
I checked out Firefox/Windows 10 and does not happen, maybe it's a Chrome quirk?

**Comment by fidgol** [comment]
I am facing the issues on Windwos10 with Firefox.
![image](https://user-images.githubusercontent.com/40062663/72877432-314eea00-3cf9-11ea-9957-89c6ba396356.png)
![image](https://user-images.githubusercontent.com/40062663/72877459-3d3aac00-3cf9-11ea-9e00-69d9372e12de.png)

**Comment by sigmasoldi3r** [comment]
True, I am able to reproduce it also maybe I did something wrong the last day :/

**Comment by sigmasoldi3r** [comment]
After some testing, I've found out that the problem has to do with _relative-absolute_ positioning that is being used with tooltip, as this stack overflow answer points out: https://stackoverflow.com/a/611098/1842548

The problem is that changing this would introduce breaking changes, as a new element must be introduced, sadly.

If we were talking inside a framework context such as React, the library consumers would not notice it, but as is a **css pure** framework, the implications are... well rather complicated.

We should look for solving this anyway.
